bitrise.yml

```diff
@@ -0,0 +1,248 @@
+
+            envman add --key JAVA_HOME --value
+            '/usr/lib/jvm/java-11-openjdk-amd64'
+    - cache-pull@2: {}
+    - install-missing-android-tools@2:
+        inputs:
+        - gradlew_path: $PROJECT_LOCATION/gradlew
+    - android-build@0:
+        inputs:
+        - variant: $INTEGRATOR_VARIANT
+        - module: $EXAMPLE_APP_MODULE
+    - android-lint@0:
+        inputs:
+        - project_location: $PROJECT_LOCATION
+        - module: $WIDGET_SDK_MODULE
+        - variant: $SDK_VARIANT
+        title: Run Lint for SDK
+    - android-unit-test@1:
+        inputs:
+        - project_location: $PROJECT_LOCATION
+        - module: $WIDGET_SDK_MODULE
+        - variant: $SDK_VARIANT
+        title: Unit Test for SDK
+    - android-lint@0:
+        inputs:
+        - project_location: $PROJECT_LOCATION
+        - module: $EXAMPLE_APP_MODULE
+        - variant: $INTEGRATOR_VARIANT
+        title: Run Lint for APP
+    - android-unit-test@1:
+        inputs:
+        - project_location: $PROJECT_LOCATION
+        - module: $EXAMPLE_APP_MODULE
+        - variant: $INTEGRATOR_VARIANT
+        title: Unit Test for APP
+    - deploy-to-bitrise-io@1:
+        inputs:
+        - notify_email_list: $BUILD_EMAILS
+    - cache-push@2: {}
+    envs:
+    - opts:
+        is_expand: false
+      INTEGRATOR_VARIANT: master
+  master:
+    steps:
+    - activate-ssh-key@4:
+        run_if: '{{getenv "SSH_RSA_PRIVATE_KEY" | ne ""}}'
+    - git-clone@4: {}
+    - cache-pull@2: {}
+    - install-missing-android-tools@2:
+        inputs:
+        - gradlew_path: $PROJECT_LOCATION/gradlew
+    - android-build@0:
+        inputs:
+        - variant: $INTEGRATOR_VARIANT
+        - module: $EXAMPLE_APP_MODULE
+    - android-lint@0:
+        inputs:
+        - project_location: $PROJECT_LOCATION
+        - module: $WIDGET_SDK_MODULE
+        - variant: $SDK_VARIANT
+        title: Run Lint for SDK
+    - android-unit-test@1:
+        inputs:
+        - project_location: $PROJECT_LOCATION
+        - module: $WIDGET_SDK_MODULE
+        - variant: $SDK_VARIANT
+        title: Unit Test for SDK
+    - android-lint@0:
+        inputs:
+        - project_location: $PROJECT_LOCATION
+        - module: $EXAMPLE_APP_MODULE
+        - variant: $INTEGRATOR_VARIANT
+        title: Run Lint for APP
+    - android-unit-test@1:
+        inputs:
+        - project_location: $PROJECT_LOCATION
+        - module: $EXAMPLE_APP_MODULE
+        - variant: $INTEGRATOR_VARIANT
+        title: Unit Test for APP
+    - browserstack-upload@0:
+        inputs:
+        - custom_id: $BROWSERSTACK_APP_ID
+        title: Upload APP to BrowserStack
+    - deploy-to-bitrise-io@1:
+        inputs:
+        - notify_email_list: $BUILD_EMAILS
+    - cache-push@2: {}
+    envs:
+    - opts:
+        is_expand: false
+      INTEGRATOR_VARIANT: master
+  publish_to_nexus:
+    description: Task builts a SDK and upload it to Nexus.
```

Comment:
I  think it existed before, but if it's not difficult for you, maybe it's better to change this description to "Task builds an SDK and uploads it to Nexus."

Reply:
Shall I do it on my own?

Reply:
Damn I didn't see this, I will open a quick PR to amend this.